Updated to Meteor 1.3 & native npm packages

NilsDannemann opened the pull request:
Added an final_updated folder, which updates the tutorial to Meteor 1.3 and native npm packages.

Had to lock the webshot npm version, since the newest seems to have some problems with bootstrap.

NilsDannemann committed:
Updated to Meteor 1.3 & native npm packages

ryanswapp commented:
Hey thanks for doing this! I'm trying to decide the best way to proceed since updating the code necessarily requires updating the written blog post. I think I'm going to wait to pull this in until I can update the blog post. Hopefully I'll have some time this weekend :)

NilsDannemann commented:
No worries. Hopefully updating it isn't too much work. 

I've literally just added the npm packages (phantomjs & webshot), replaced require with imports and moved the method out of the isServer block and put it in the server-folder instead. Oh, and I had to lock the webshot npm version. Everything else exactly like you created it :)
